### PR TITLE
Everywhere: Merge the WebSocket service into RequestServer

### LINKED ISSUE #23080: Merge WebSocket and RequestServer services
I don't see a good reason at the moment to keep them as separate processes. Both are handling untrusted network data. We don't have a separate service for HTTP/HTTPS/Gemini/etc that are supported by RequestServer.

We should merge WebSocket handling into the same model that RequestServer does and just have one flavor of 'NetworkHelper' process.

### PR DESCRIPTION
**Pull request by alimpfard**
This keeps the APIs separate as they are wildly different, a future improvement could be to somehow unify the APIs (if possible).

Closes #23080.

### CODE REVIEW

File: Ladybird/Android/src/main/cpp/WebSocketService.cpp [hunk 1]
```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2021, Dex♪ <dexes.ttp@gmail.com>
- * Copyright (c) 2023, Andrew Kaster <akaster@serenityos.org>
- *
- * SPDX-License-Identifier: BSD-2-Clause
- */
-
-#include <AK/LexicalPath.h>
```
Comment (ADKaster) [inline Ladybird/Android/src/main/cpp/WebSocketService.cpp:8]: there's more that can be deleted here, such as

WebSocketService.kt
`bindWebSocket` in WebContentService.kt
extracting `bindWebSocket` jmethodID in WebContentServiceJNI.cpp
`bind_web_socket_java` function.
 `bind_web_socket_service` in WebContentService.cpp

Comment (alimpfard) [reply Ladybird/Android/src/main/cpp/WebSocketService.cpp:8]: (pls unresolve if you find other things to remove)

**Comment by ADKaster** [comment]
lgtm, nice to see that negative diff :D. 

Next yakbait is to see if you'll do the work of making this boi single-instance, managed/owned by the chrome process and handing out sockets to WebContent and WebWorker processes as needed.

**Review by ADKaster — COMMENTED** [review]
Base/home/anon/.config/SystemServer.ini
Ladybird/cmake/InstallRules.cmake 
    `foreach service IN LISTS webcontent requestserver websocket webworker`

**Pull request merged**